integration_tests/support/state.ts

```diff
@@ -0,0 +1,13 @@
+let smokeTestBarcode: string
+const setSmokeTestBarcode = val => {
+  smokeTestBarcode = val
+  return null
+}
+const getSmokeTestBarcode = (): string => {
+  return smokeTestBarcode
+}
+
+export default {
+  setSmokeTestBarcode,
+  getSmokeTestBarcode,
```

Comment:
Required to save the generated barcode between tests while remaining async

Reply:
Nice, that is easy to grok and follow 👍